collect-settings.rst

```diff
@@ -125,7 +171,10 @@ You can also import settings from a QR code saved to the device, by selecting :g
 
 .. _user-access-control-settings:
 
-User Access Control Settings
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+.. rubric:: User Access Control Settings
```

Comment:
why was this changed to a rubric?

Reply:
in response to your question:

For this to be a rubric would suggest it is a part of the Import/Export Settings section.
This one should have stayed a heading, at the same level as Import/Export settings (~~~~~~~~)

The other ones make sense as rubrics, as they are consistent.

Reply:
Oh this was an accident. sorry sir.